loadtest/control/simulcontroller/actions.go

```diff
@@ -139,11 +148,16 @@ func (c *SimulController) loginOrSignUp(u user.User) control.UserActionResponse
 }
 
 func (c *SimulController) login(u user.User) control.UserActionResponse {
+	start := time.Now().UnixMilli()
 	for {
 		resp := control.Login(u)
 		if resp.Err == nil {
 			err := c.connect()
 			if err == nil {
+				err := c.user.ObserveClientMetric(model.ClientTimeToFirstByte, float64(time.Now().UnixMilli()-start)/1000)
+				if err != nil {
+					mlog.Warn("Failed to store observation", mlog.Err(err))
+				}
```

Comment:
TIL! We want to measure that whenever a user logs in (thinking of a scenario where opens a web page). But the user could also be logged in with the session so we may need to cover that scenario. The goal of the PR is to measure the load in the server with client metrics, but I think it could be a good idea to actually get realistic metrics from the agents. 

Maybe we can add this to backlog, WDYT?